packages/editor-ui/src/components/executions/workflow/WorkflowExecutionsSidebar.vue

```diff
@@ -117,6 +119,12 @@ export default defineComponent({
 	data() {
 		return {
 			filter: {} as ExecutionFilterType,
+			mountedItems: [] as string[],
```

Comment:
Looks like we only care about number of mounted cards here, so can we just track that instead of keeping the list of ids (I guess it will be a bit more performant for longer lists)?

Reply:
Yes, it could be just a counter but I need to get a workflow card ref by its ID and since it is used in other places (it is needed for the auto scroll) I didn't want to change the ref so it seemed better to collect the IDs
![CleanShot 2024-05-27 at 12 19 57](https://github.com/n8n-io/n8n/assets/5410822/f94950ef-3a37-4fe4-80b8-c66a86a7c716)

I don't think it will cause a performance issue, you would have to have like 100K (or a million) of IDs stored in the array, I don't think we will ever have that